assets/app/views/settings.html

```diff
@@ -175,6 +199,6 @@ <h3 class="text-center">Memory</h3>
           </table>
         </div>
       </div>
-    </div>
+
```

Comment:
Yup... `build-assets.sh` wouldn't build w/o that.  I broke the 'edit settings' chunk out into a partial include to make it a bit easier to work through all the tags.  Tempted to do the same with the other components as well...